packages/actor-http-fetch/lib/ActorHttpFetch.ts

```diff
@@ -92,31 +91,30 @@ export class ActorHttpFetch extends ActorHttp {
 
   public async run(action: IActionHttp): Promise<IActorHttpOutput> {
     // Prepare headers
-    const initHeaders = action.init?.headers ?? {};
-    action.init = action.init ?? {};
-    action.init.headers = new Headers(initHeaders);
-    if (!action.init.headers.has('user-agent')) {
-      action.init.headers.append('user-agent', this.userAgent);
+    const headers = new Headers(action.init?.headers);
+    if (!headers.has('user-agent')) {
+      headers.set('user-agent', this.userAgent);
     }
-    const authString: string | undefined = action.context.get(KeysHttp.auth);
+    const authString = action.context.get<string>(KeysHttp.auth);
     if (authString) {
-      action.init.headers.append('Authorization', `Basic ${Buffer.from(authString).toString('base64')}`);
+      headers.set('authorization', `basic ${Buffer.from(authString).toString('base64')}`);
```

Comment:
Yes, because the `user-agent` header was already lowercase here, and it seems like the `Headers` object maintains the case somehow (so the expected `toHaveBeenCalledWith` header case needs to match the case provided here).

I will change both to use uppwercase, though, just in case, so that the `Authorization` header remains the same.

Reply:
"Don't touch what ain't broken" :-)